config/consensus.go

```diff
@@ -516,6 +516,18 @@ type ConsensusParams struct {
 	// used by agreement for Circulation, and updates the calculation of StateProofOnlineTotalWeight used
 	// by state proofs to use the same method (rather than excluding stake from the top N stakeholders as before).
 	ExcludeExpiredCirculation bool
+
+	// DynamicFilterTimeout specifies whether the timeout for the filter step
+	// should be determined dynamically, at run-time.
+	DynamicFilterTimeout bool
```

Comment:
What's the difference between `DynamicFilterTimeout=false` and `DynamicFilterPayloadArriavalHistory=0`?

Reply:
You're right. The flag is redundant, I'll remove it.